scheds/rust/scx_p2dq/src/bpf/main.bpf.c

```diff
@@ -1864,6 +1905,26 @@ static int init_llc(u32 llc_index)
 		return ret;
 	}
 
+	// Initialize CPU sharding fields
+	llcx->nr_shards = p2dq_config.llc_shards;
+
+	if (p2dq_config.llc_shards > 0) {
+		llcx->nr_shards = min(p2dq_config.llc_shards, MAX_LLC_SHARDS);
```

Comment:
Should we also limit nr_shards to the amount of CPUs in the LLC?

Reply:
Yeah, that makes sense